backend/clubs/views.py

```diff
@@ -2878,33 +2878,74 @@ def get(self, request):
                                     type: integer
         ---
         """
-        id = request.query_params.get("event")
-        if not id:
-            return Response({"count": 0})
-
-        try:
-            conn = get_redis_connection()
-        except NotImplementedError:
-            return Response({"count": 0})
-        key = f"zoom:meeting:live:{id}"
-        ans = conn.get(key)
-        if ans is None:
-            ans = 0
-        else:
-            ans = int(ans)
-        return Response({"count": ans})
+        meeting_id = request.query_params.get("event")
+        return Response({"count": ZoomMeetingVisit.objects.filter(meeting_id=meeting_id).count()})
 
     def post(self, request):
         action = request.data.get("event")
-        meeting_id = request.data.get("payload", {}).get("object", {}).get("id", None)
-        if meeting_id is not None:
-            conn = get_redis_connection()
-            key = f"zoom:meeting:live:{meeting_id}"
-            if action == "meeting.participant_joined":
-                conn.incr(key)
-            elif action == "meeting.participant_left":
-                conn.decr(key)
-            conn.expire(key, datetime.timedelta(hours=8))
+        if action == "meeting.participant_joined":
+            email = (
+                request.data.get("payload", {})
+                .get("object", {})
+                .get("participant", {})
+                .get("email", None)
+            )
+
+            person = None
+            if email:
+                username = email.split("@")[0]
+                res = get_user_model().objects.filter(username=username)
+                if res:
+                    person = res.first()
+
+            meeting_id = request.data.get("payload", {}).get("object", {}).get("id", None)
+            event = Event.objects.filter(url__contains=meeting_id).first()
```

Comment:
I'd recommend using some regex to make this more precise (see `__regex` for details)!

Reply:
Ok great tip I will check that out!